package.json

```diff
@@ -44,20 +48,18 @@
     "deploy:staging:android": "yarn prepare:staging && eas build --platform android --profile staging --auto-submit --non-interactive",
     "deploy:staging:ios": "yarn prepare:staging && eas build --platform ios --profile staging --auto-submit --non-interactive",
     "deploy:staging": "yarn prepare:staging && eas build --platform all --profile staging --auto-submit --non-interactive",
-    "eas-build-pre-install": "base64 --help && echo $ANDROID_FIREBASE_CONFIG | base64 --decode > google-services.json  && cat google-services.json && echo $IOS_FIREBASE_CONFIG | base64 --decode > GoogleService-Info.plist && cat GoogleService-Info.plist",
-    "generate:component": "node ./scripts/create_new_component.js && yarn eslint src --fix && yarn tsc",
+    "eas-build-pre-install": "base64 --help && echo $ANDROID_FIREBASE_CONFIG | base64 --decode > google-services.json && cat google-services.json && echo $IOS_FIREBASE_CONFIG | base64 --decode > GoogleService-Info.plist && cat GoogleService-Info.plist",
     "generate:env:production": "scripts/generate_dotenv.sh production",
     "generate:env:qa": "scripts/generate_dotenv.sh qa",
     "generate:env:staging": "scripts/generate_dotenv.sh staging",
     "generate:google-services-config": "./scripts/generate_firebase_config.sh",
     "generate:icon:types": "node ./scripts/generate_icon_types.js",
```

Comment:
`generate:icon:types` script does not work. The file is moved

Reply:
Removed!